src/App.js

```diff
@@ -39,12 +42,14 @@ function App() {
         {isLoading ? <WeatherLoader /> : null}
 
         <h1>Weather App</h1>
-
-        <input
-          type="text"
-          placeholder="Enter city/town..."
-          onChange={(e) => setSearch(e.target.value)}
-        />
+        <Input>
+          <input
```

Comment:
styled components are used for styling html tags so you dont need <input/> here since you created <Input/> component.